src/Halcyon/Processors/SectionParser.php

```diff
@@ -284,7 +284,7 @@ private static function calculateLinePosition(string $content, int $instance = 1
             }
 
             if ($count === $instance) {
-                return static::adjustLinePosition($content, $number);
+                return self::adjustLinePosition($content, $number);
```

Comment:
instead of changing this to self the method should be changed to protected and this should remain static. The other private method should be made protected too.